src/components/notifications/AppNotifications/AppNotificationsHeader/index.tsx

```diff
@@ -25,7 +29,7 @@ const AppNotificationsHeader: React.FC<IAppNotificationsHeaderProps> = ({
 }) => {
   const isMobile = useIsMobile()
   const { dappOrigin } = useContext(W3iContext)
-  const [dropdownToShow, setDropdownToShow] = useState<string | undefined>()
+  const url = new URL('https://' + domain)
```

Comment:
Definitely not happy with this. We don't have full URL of the dApp in the metadata from Notify Client. 

@devceline Do you think we could do this better? Maybe like matching, the dApp with projects and finding the project object, and getting the URL? Both looking overengineering but we cannot update the Notify Client for this too.

Reply:
Let's move the projects to a Context. We can get the project list in the AppCard find the project and use the URL from the project object. That would be a better approach. I wouldn't want to ship this. I'll take a look later

Reply:
Can we huddle, don't fully understand this here?

Reply:
Okay let's go with this way since we are already sure that the domain wouldn't be empty as we check. I'd still like to have url from the backend instead of creating it on the UI